benches/nexmark/generator/mod.rs

```diff
@@ -35,6 +35,14 @@ pub struct NexmarkGenerator<R: Rng> {
     wallclock_base_time: Option<u64>,
 }
 
+pub fn wallclock_time() -> Result<u64> {
+    let millis: u64 = SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)?
```

Comment:
You can unwrap this first error, `SystemTime::now()` will never be before `UNIX_EPOCH`